### PR TITLE
Presence: RealLifeGTAV

### PR DESCRIPTION
**Pull request by N0chteil**

### CODE REVIEW

File: RealLifeGTAV/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  "description": {
+    "en": "Real Life GTAV is a german FiveM roleplay server with custom scripts and more!",
+	"de": "Real Life GTAV ist ein deutscher FiveM Roleplay Server mit eigenen Scripts und mehr!"
+  },
+  "url": [
+    "reallifegtav.de",
+    "www.reallifegtav.de"
+  ],
+  "version": "1.0.0",
+  "logo": "https://reallifegtav.de/images/logo.png",
```
Comment (ririxichan) [inline RealLifeGTAV/dist/metadata.json:16]: should be imgur link like thumbnail

File: RealLifeGTAV/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  "thumbnail": "https://i.imgur.com/QjWhm3D.png",
+  "color": "black",
```
Comment (ririxichan) [inline RealLifeGTAV/dist/metadata.json:18]: need to be hex

File: RealLifeGTAV/presence.ts [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+		}
+	}
+  }
+  if (presenceData.details == null) {
+    presence.setTrayTitle();
+    presence.setActivity();
+  } else {
+    presence.setActivity(presenceData);
+  }
+});
```
Comment (ririxichan) [inline RealLifeGTAV/presence.ts:73]: fix tabs and spaces.

**Review by ririxichan — CHANGES_REQUESTED** [review]
Upload proof of working presence and check comments.

File: RealLifeGTAV/presence.ts [hunk 4]
```diff
@@ -0,0 +1,73 @@
+const presence = new Presence({
+  clientId: "760863245224640555"
+});
+
+const browsingStamp = Math.floor(Date.now() / 1000);
```
Comment (Bas950) [inline RealLifeGTAV/presence.ts:5]: merge consts

**Review by Bas950 — CHANGES_REQUESTED** [review]
And fix all the issues from cadacy or that ririxichan specified.

File: RealLifeGTAV/presence.ts (hunk 4 above)
Comment (N0chteil) [reply RealLifeGTAV/presence.ts:5]: Have I merged it? Everything here confuses me xD

Comment (ririxichan) [reply RealLifeGTAV/presence.ts:5]: You didn't just push commits to your repo master branch.

Comment (Bas950) [reply RealLifeGTAV/presence.ts:5]: Merge consts:
const var1 = obj,
var2 = obj;

File: RealLifeGTAV/dist/metadata.json [hunk 5]
```diff
@@ -13,9 +13,9 @@
     "www.reallifegtav.de"
   ],
   "version": "1.0.0",
-  "logo": "https://reallifegtav.de/images/logo.png",
+  "logo": "https://i.imgur.com/upst6fX.png",
   "thumbnail": "https://i.imgur.com/QjWhm3D.png",
```
Comment (Bas950) [inline RealLifeGTAV/dist/metadata.json]: logo needs to be square and 512x512 just like the one uploaded to the discord app

**Review by Bas950 — CHANGES_REQUESTED** [review]

**Comment by Bas950** [comment]
Also upload proof of it working

File: RealLifeGTAV/presence.ts (hunk 4 above)
Comment (N0chteil) [reply RealLifeGTAV/presence.ts:5]: What should I do with it? (I've never done anything like that before :/)

Comment (N0chteil) [reply RealLifeGTAV/presence.ts:5]: **_Pending_** what have I done now xD

File: RealLifeGTAV/dist/metadata.json (hunk 5 above)
Comment (ririxichan) [reply RealLifeGTAV/dist/metadata.json]: still not 512x512

File: RealLifeGTAV/presence.ts (hunk 4 above)
Comment (ririxichan) [reply RealLifeGTAV/presence.ts:5]:
```javascript
const presence = new Presence({
  clientId: "760863245224640555"
}),
 browsingStamp = Math.floor(Date.now() / 1000);
```

File: RealLifeGTAV/dist/metadata.json [hunk 6]
```diff
@@ -0,0 +1,25 @@
+{
+  "author": {
+    "name": "N0chteil ^^",
+    "id": "495901098926669825"
+  },
+  "service": "Real Life GTAV",
+  "description": {
+    "en": "Real Life GTAV is a german FiveM roleplay server with custom scripts and more!",
+	"de": "Real Life GTAV ist ein deutscher FiveM Roleplay Server mit eigenen Scripts und mehr!"
```
Comment (ririxichan) [inline RealLifeGTAV/dist/metadata.json:9]: fix this tab too.

**Review by ririxichan — CHANGES_REQUESTED** [review]
upload proof of working and fix it.

File: RealLifeGTAV/dist/metadata.json (hunk 5 above)
Comment (N0chteil) [reply RealLifeGTAV/dist/metadata.json]: https://prnt.sc/uqsvt0 512x512

File: RealLifeGTAV/dist/metadata.json (hunk 6 above)
Comment (ririxichan) [reply RealLifeGTAV/dist/metadata.json:9]: still not fixed tabs.

Comment (N0chteil) [reply RealLifeGTAV/dist/metadata.json:9]: I see no difference between the current one.

Comment (ririxichan) [reply RealLifeGTAV/dist/metadata.json:9]: ```	"de":``` there are 2 tabs instead of one like before ```"en":```

**Comment by N0chteil** [comment]
> upload proof of working and fix it.

How should I do that?
Here (https://docs.premid.app/en/dev/presence) i didn't find anything
(i'm stupid, i know xD)

**Comment by ririxichan** [comment]
Just screenshot presence when you use it. Just like this: 
image
We need to know if it is working.
image

**Comment by N0chteil** [comment]
> Just screenshot presence when you use it. Just like this: 
> image
> We need to know if it is working.
> image
> 

In the browser extension?

File: RealLifeGTAV/dist/metadata.json [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+  "author": {
+    "name": "N0chteil ^^",
+    "id": "495901098926669825"
+  },
+  "contributors": [
+    {
+      "name": "N0chteil",
+      "id": "495901098926669825"
+    }
+  ],
```
Comment (Bas950) [inline RealLifeGTAV/dist/metadata.json:12]: remove

File: RealLifeGTAV/dist/metadata.json [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+  "url": [
+    "reallifegtav.de",
+    "www.reallifegtav.de"
+  ],
+  "description": {
+    "en": "Real Life GTAV is a german FiveM roleplay server with custom scripts and more!",
+	  "de": "Real Life GTAV ist ein deutscher FiveM Roleplay Server mit eigenen Scripts und mehr!"
+  },
+  "service": "Real Life GTAV",
+  "version": "1.0.0",
```
Comment (Bas950) [inline RealLifeGTAV/dist/metadata.json:22]: service name and folder name must be 100% the same

File: RealLifeGTAV/dist/metadata.json [hunk 9]
```diff
@@ -0,0 +1,34 @@
...
+  "contributors": [
+    {
+      "name": "N0chteil",
+      "id": "495901098926669825"
+    }
+  ],
+  "url": [
+    "reallifegtav.de",
+    "www.reallifegtav.de"
+  ],
```
Comment (Bas950) [inline RealLifeGTAV/dist/metadata.json:16]: probs only need 1 of those

**Review by Bas950 — CHANGES_REQUESTED** [review]
Fix codacy errors

File: RealLifeGTAV/dist/metadata.json (hunk 9 above)
Comment (N0chteil) [reply RealLifeGTAV/dist/metadata.json:16]: ok

File: RealLifeGTAV/dist/metadata.json (hunk 8 above)
Comment (N0chteil) [reply RealLifeGTAV/dist/metadata.json:22]: ok

File: RealLifeGTAV/dist/metadata.json (hunk 7 above)
Comment (N0chteil) [reply RealLifeGTAV/dist/metadata.json:12]: ok

File: RealLifeGTAV/dist/metadata.json [hunk 10]
```diff
@@ -4,31 +4,22 @@
...
+  "service": "RealLifeGTAV",
   "description": {
     "en": "Real Life GTAV is a german FiveM roleplay server with custom scripts and more!",
-	  "de": "Real Life GTAV ist ein deutscher FiveM Roleplay Server mit eigenen Scripts und mehr!"
+    "de": "Real Life GTAV ist ein deutscher FiveM Roleplay Server mit eigenen Scripts und mehr!"
   },
-  "service": "Real Life GTAV",
+  "url": [
+    "reallifegtav.de"
+  ],
```
Comment (Bas950) [inline RealLifeGTAV/dist/metadata.json:14]: doesnt need to be in an array if its only 1

**Review by Bas950 — CHANGES_REQUESTED** [review]

File: RealLifeGTAV/dist/metadata.json (hunk 10 above)
Comment (N0chteil) [reply RealLifeGTAV/dist/metadata.json]: fixed, thanks.

**Comment by N0chteil** [comment]
Screenshots: 
Screenshot_548
Screenshot_549
Screenshot_550
Screenshot_551

File: RealLifeGTAV/presence.ts [hunk 11]
```diff
@@ -0,0 +1,89 @@
+var presence = new Presence({
+    clientId: "760863245224640555"
+}), browsingStamp = Math.floor(Date.now() / 1000);
```
Comment (Bas950) [inline RealLifeGTAV/presence.ts:3]: no var, use const

File: RealLifeGTAV/presence.ts [hunk 12]
```diff
@@ -0,0 +1,89 @@
...
+
+presence.on("UpdateData", function () {
+    var presenceData = {
+        largeImageKey: "reallifegtav"
+    };
```
Comment (Bas950) [inline RealLifeGTAV/presence.ts:8]: no var, use const

**Review by Bas950 — CHANGES_REQUESTED** [review]
Fix codacy errors

File: RealLifeGTAV/presence.ts (hunk 12 above)
Comment (N0chteil) [reply RealLifeGTAV/presence.ts]: edited

**Comment by Bas950** [comment]
Also upload proof of it working

> Screenshots:
> Screenshot_548
> Screenshot_549
> Screenshot_550
> Screenshot_551

remove premid out of the name

**Comment by Bas950** [comment]
Upload a screenshot from the discord website then that it says it without premid in the name

**Comment by N0chteil** [comment]
> Upload a screenshot from the discord website then that it says it without premid in the name

image

**Review by Bas950 — APPROVED** [review]
Fix codacy errors

**Review by Timeraa — APPROVED** [review]

**Pull request merged**